packages/design-tokens/package.json

```diff
@@ -34,10 +34,9 @@
     "prop-types": "^15.7.2"
   },
   "devDependencies": {
-    "@types/jest": "^28.1.6",
+    "@types/jest": "^29.0.0",
     "@types/prop-types": "^15.7.5",
-    "jest": "^28.1.3",
-    "jest-extended": "^3.0.2",
```

Comment:
We don't need this, the design-tokens packages has two tests (snapshots) and using the built-in `toEqual` matcher works fine.